recipes/git-hooks/pre-commit-phpcs/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Run pre-commit check PHP script inside ddev when committing from host.
+if [ "$IS_DDEV_PROJECT" != true ]; then
+  ddev exec /usr/bin/php scripts/git/pre-commit-phpcs.php
+else
+  /usr/bin/php scripts/git/pre-commit-phpcs.php
```

Comment:
Is `php` always run from this location on a host? 
Maybe change to below so it can be picked up from $PATH

```
  php scripts/git/pre-commit-phpcs.php
```

Reply:
On most systems it will be in PATH. Same goes for `ddev exec`, it can just be `ddev exec php`

Reply:
I was concerned about `else` version more so because of the variety of hosts (Win, WSL, Linux, MacOS).

Reply:
@tyler36 the `else` part means you are inside the container, so it will be always a ddev container (Linux) and `/usr/bin/php` is the right path.

Remember, this script does not use host php, it detects whether you are committing from the host or from the container.

Reply:
You're right. 

Sorry for the confusion. Still waiting for the coffee to kick in.

Reply:
No worries. I'll resolve the conversation then.

Please let Randy know if this is ok now so that it gets merged. An extra check to the documentation would be great if you have the time. I was careful, but you never know.